### ISSUE TITLE
Getting this error every initial queueing of a prompt

### ISSUE THREAD

**Issue by KariteDono** [comfyanonymous/ComfyUI#3341]
got prompt
model_type EPS
Using pytorch attention in VAE
Using pytorch attention in VAE
clip missing: ['clip_l.logit_scale', 'clip_l.transformer.text_projection.weight']
Requested to load SDXLClipModel
Loading 1 new model
C:\Users\alecd\Downloads\AI\new_ComfyUI_windows_portable_nvidia_cu121_or_cpu\ComfyUI_windows_portable\ComfyUI\comfy\ldm\modules\attention.py:345: UserWarning: 1Torch was not compiled with flash attention. (Triggered internally at ..\aten\src\ATen\native\transformers\cuda\sdp_utils.cpp:455.)
  out = torch.nn.functional.scaled_dot_product_attention(q, k, v, attn_mask=mask, dropout_p=0.0, is_causal=False)

--I'm not sure if the clip issue and the flash attention issue are correlated but after this error the image pops up as just white noise, I've tried updating the dependencies, using the .bat update file, reinstalling comfyui, adding comfyui file to my system environment path, reinstalling torch, and updating python with no success any help would be appreciated.

**Comment by Akira13641** [comment]
I think the Windows versions of Pytorch for some reason are straight up not compiled with flash attention above version 2.1.2, it's not clear why Comfy started bundling higher versions without seeming to take that into account

**Comment by KariteDono** [comment]
Gotcha when I try to uninstall torch to do a roll back I get this error when running the install command and the uninstall - Fatal error in launcher: Unable to create process using '"D:\a\ComfyUI\python_embeded\python.exe"  "C:\Users\alecd\Downloads\AI\new_ComfyUI_windows_portable_nvidia_cu121_or_cpu\ComfyUI_windows_portable\python_embeded\Scripts\pip.exe" uninstall torch': The system cannot find the file specified.

I've looked through the files and am a bit confused on where to go from here.

**Comment by Akira13641** [comment]
I just rechecked the previous versions section on the PyTorch website, this is the command they give, which IIRC worked / works for me:

`pip install torch==2.1.2 torchvision==0.16.2 torchaudio==2.1.2 --index-url https://download.pytorch.org/whl/cu121`

**Comment by Akira13641** [comment]
> Gotcha when I try to uninstall torch to do a roll back I get this error when running the install command and the uninstall - Fatal error in launcher: Unable to create process using '"D:\a\ComfyUI\python_embeded\python.exe" "C:\Users\alecd\Downloads\AI\new_ComfyUI_windows_portable_nvidia_cu121_or_cpu\ComfyUI_windows_portable\python_embeded\Scripts\pip.exe" uninstall torch': The system cannot find the file specified.
> 
> I've bypassed the error above by using python -m and uninstalled torch but now trying to install an older version get this error: ERROR: Could not find a version that satisfies the requirement torch==2.1.2 (from versions: 2.2.0, 2.2.0+cpu, 2.2.0+cu118, 2.2.0+cu121, 2.2.1, 2.2.1+cpu, 2.2.1+cu118, 2.2.1+cu121, 2.2.2, 2.2.2+cpu, 2.2.2+cu118, 2.2.2+cu121, 2.3.0, 2.3.0+cpu, 2.3.0+cu118, 2.3.0+cu121)

I just rechecked the previous versions section on the PyTorch website, this is the command they give, which IIRC worked / works for me:

`pip install torch==2.1.2 torchvision==0.16.2 torchaudio==2.1.2 --index-url https://download.pytorch.org/whl/cu121`

**Comment by Akira13641** [comment]
The index url there is the key, I guess

**Comment by KariteDono** [comment]
I got it to work. The python I had installed on the system was the one being interpreted when using the -m command I uninstalled 3.12 and reinstalled 3.11 so it would recognize the 311 .whl files from pytorch's website thank you for the help.
